test/guest/cockpit-fedora-23.setup

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash
+
+set -x
+
+echo foobar | passwd --stdin root
+
+# Let's not bother with SELinux until f23 is released.
+#
+# Known problems:
+#   https://bugzilla.redhat.com/show_bug.cgi?id=1250842
+#
+setenforce 0
+ln -sf ../selinux/config /etc/sysconfig/selinux
+echo 'SELINUX=permissive' > /etc/selinux/config
+
+# We install all dependencies of the cockpit packages since we want
+# them to not spontaneously change from one test run to the next when
+# the distribution repository is updated.
+#
+COCKPIT_DEPS="json-glib realmd glib-networking libssh selinux-policy-targeted docker pcp-libs pcp kubernetes etcd subscription-manager storaged storaged-lvm2 device-mapper-multipath"
+
+# We also install the packages necessary to join a FreeIPA domain so
+# that we don't have to go to the network during a test run.
+#
+IPA_CLIENT_PACKAGES="freeipa-client oddjob oddjob-mkhomedir sssd"
+
+TEST_PACKAGES="systemtap-runtime-virtguest"
+
+rm -rf /etc/sysconfig/iptables
+
+maybe() { if type "$1" >/dev/null 2>&1; then "$@"; fi; }
+
+# For the D-Bus test server
+maybe firewall-cmd --permanent --add-port 8765/tcp
+
+echo 'NETWORKING=yes' > /etc/sysconfig/network
+
+useradd -u 1000 -c Administrator -G wheel admin
+echo foobar | passwd --stdin admin
+
+# Let's upgrade dnf first and then use the new dnf to upgrade
+# the rest. This helps avoid things like:
+#
+#     https://bugzilla.redhat.com/show_bug.cgi?id=1211547
+#
+dnf -y upgrade dnf
+
+dnf -y upgrade
+dnf -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
+
+rm -rf /var/log/journal/*
```

Comment:
this needs to move to the end of setup like for the other configurations in #2758